package.json

```diff
@@ -92,7 +92,7 @@
     "@rollup/plugin-replace": "^5.0.5",
     "@rollup/plugin-terser": "^0.4.4",
     "@storybook/react": "^6.5.0-beta.8",
-    "@stripe/stripe-js": "^7.8.0",
+    "@stripe/stripe-js": "file:/Users/pololi/stripe/stripe-js/dist",
```

Comment:
stripe-js is NOT published yet. So can only do it locally. Will update it once it's published. https://github.com/stripe/stripe-js/pull/803